src/flow/wernerwengle_mod.F90

```diff
@@ -82,4 +82,34 @@ PURE ELEMENTAL REAL(realk) FUNCTION tauwin(uquer, dds)
             tauwin  = vz*2.0*gmol*uquern/dds
         END IF
     END FUNCTION tauwin
+
+
+    PURE ELEMENTAL REAL(realk) FUNCTION qwallfix(tbound, tfluid, uquer, dds, &
```

Comment:
Maybe we should add that this is not tested and validated at all... Or did you conduct any tests with this implementation?

Reply:
No, we did not test anything... And in the immediate future we have no plans for anything that require scalar transport either.